.github/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
I wonder if we need this template at all? GH provides a [Open a regular issue](https://github.com/SquidDev-CC/CC-Tweaked/issues/new/choose) button, which may be sufficient.

Reply:
Ah, didn't realize that GH provided a fallback.  That should work.